services/queue/src/main.js

```diff
@@ -226,6 +229,10 @@ let load = loader({
         workerInfo: ctx.workerInfo,
         LRUcache: new QuickLRU({ maxSize: ctx.cfg.app.taskCacheMaxSize || 10 }),
         objectService: ctx.objectService,
+        taskMaxDependencies: Math.min(
+          ctx.cfg.app.taskMaxDependencies || DEFAULT_MAX_TASK_DEPENDENCIES,
```

Comment:
```suggestion
          ctx.cfg.app.taskMaxDependencies ?? DEFAULT_MAX_TASK_DEPENDENCIES,
```
If `ctx.cfg.app.taskMaxDependencies` is intentionally 0, i think `||` would consider that false and `taskMaxDependencies` would end up being `DEFAULT_MAX_TASK_DEPENDENCIES`, rather than 0. Probably not a real use case, as probably `ctx.cfg.app.taskMaxDependencies` should never be 0, but if it was, I think `??` would be the correct operator to have `taskMaxDependencies` be 0 if `ctx.cfg.app.taskMaxDependencies` actually is 0. But I'm no javascript expert, and maybe I'm misunderstanding.

Reply:
Oh! Great eagle-eye! I should probably rewrite it though, as node is "special" 

```
> let a
undefined
> a
undefined
> a || 5
5
> a = 0
0
> a || 5
5
> undefined || 'a'
'a'
> 0 || 'a'
'a'
> 0 || 5
5
```

Looks like `||` works differently for numbers 😬

Reply:
Indeed! Thanks for catching this!